packages/roosterjs-editor-types/lib/enum/ExperimentalFeatures.ts

```diff
@@ -3,27 +3,27 @@
  */
 export const enum ExperimentalFeatures {
     /**
-     * Toggle indentation using VList
+     * @deprecated This feature is always enabled
```

Comment:
We don't remove the graduated feature flights?

Reply:
No we don't. This is for backward compability.

We need to bump up major version if we remove element of public interface since it may break the build when others are still using it in their code.

Reply:
okay